### PR TITLE
UI & UX Finalization

### LINKED ISSUE #56: Decrease waiting time for transitions
_Nursultan Askarbekuly, [4/23/2022 9:47 AM]_
> There shouldn’t be much waiting when I press an option. There should be some waiting for the answer to my phrase, but also not too long.

https://user-images.githubusercontent.com/11016151/165112826-92081c4e-91e4-4c1e-bdbb-72eb82f2f971.mp4

### LINKED ISSUE #57: Ensure links work in the markdown renderer
_Nursultan Askarbekuly, [4/23/2022 9:49 AM]_
> The link is not clickable

link

### LINKED ISSUE #58: Investigate the unresponsive transition in the Innopolis story
_Nursultan Askarbekuly, [4/23/2022 9:55 AM]_
> The first button does not work, even though it is a transition

https://user-images.githubusercontent.com/11016151/165113358-ae733a7f-9e5f-4cdf-8f63-2d2cbd9da480.mp4

### LINKED ISSUE #59: Don't show the "confirm exit" dialog if nothing will be lost
_Nursultan Askarbekuly, [4/23/2022 9:58 AM]_
> The exit dialogue is confusing, I’ve just uploaded and then it tells me that I’ll loose the progress. Same dialogue appears when I finish a story and want to leave. When a user completed something we should give them a way to leave gracefully and fulfilled with the achievement

https://user-images.githubusercontent.com/11016151/165113568-7c8754cb-507f-4d36-980f-fc083728994a.mp4

### PR DESCRIPTION
**Pull request by alkaitagi**
#### Translation

- [x] Split editor into pages
- [x] Allow play-testing

#### Home

- [x] Story-screen no card
- [x] Story card text on top

#### Profile

- [x] Stories shelfs for profile lists
- [x] Sheeted expanded profile lists

#### Miscellaneous

- [x] Closes #56
- [x] Closes #57
- [x] Closes #58
- [x] Closes #59

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -7,21 +7,21 @@ packages:
...
-    version: "38.0.0"
+    version: "39.0.0"
   algolia:
     dependency: "direct main"
     description:
       name: algolia
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.0.5+1"
+    version: "1.1.0+2"
```
Comment (aabounegm) [inline pubspec.lock:17]: Can we roll this back to v1.0.5 until they fix the web support? It doesn't seem that they added any new useful features.

Comment (aabounegm) [reply pubspec.lock:17]: Or even better, seems like they just published v1.1.1 around an hour ago, supposedly fixing web support, but it hasn't been analyzed by pub yet.

File: lib/modules/translation/screens/translations.dart [hunk 2]
```diff
@@ -67,41 +95,117 @@ class TranslationEditorState extends State<TranslationEditorScreen> {
...
+                controller: _paging,
+                physics: const NeverScrollableScrollPhysics(),
                 children: [
-                  ListTile(
-                    leading: const Icon(Icons.language_rounded),
-                    title: Text(target.language),
+                  ListView(
+                    padding: const EdgeInsets.only(bottom: 76),
+                    children: [
+                      // ignore: prefer_const_constructors
```
[the remark this answers is not in the document]
Comment (alkaitagi) [reply lib/modules/translation/screens/translations.dart:118]: I've mentioned it on other PR, can't find right now. These widgets don't have state, they track parent's via context. Flutter however can't see this connection, and suggest to const the widgets. In which case they become freeze and don't get updated by context watch.

[the remark this answers is not in the document]
Comment (aabounegm) [reply lib/modules/translation/screens/translations.dart:118]: In that case, isn't it better to remove the `const` constructor itself and instead ignore the `prefer_const_constructors_in_immutables` rule in just the line above the constructor?

**Review by aabounegm — APPROVED** [review]

File: lib/modules/play/widgets/audio_slider.dart [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
 import 'package:provider/provider.dart';
 
 import '../utils/animator.dart';
 import '../utils/audio_controller.dart';
 
 class AudioSlider extends StatefulWidget {
-  const AudioSlider(this.node, {Key? key}) : super(key: key);
+  const AudioSlider(
+    this.url, {
+    this.collapsable = true,
```
Comment (aabounegm) [inline lib/modules/play/widgets/audio_slider.dart:10]: *_collapsible_

**Review by aabounegm — APPROVED** [review]

**Pull request merged**